src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CEO {
+    private String name;
+    private List<Manager> managers;
+
+    public CEO(String name) {
+        this.name = name;
+        this.managers = new ArrayList<Manager>();
```

Comment:
![image](https://user-images.githubusercontent.com/18233796/38467412-72a529b6-3b38-11e8-9b3f-8a045d5b581e.png)
`this.managers = new ArrayList<>();`